src/main/java/bowling/state/Ready.java

```diff
@@ -0,0 +1,30 @@
+package bowling.state;
+
+import bowling.pin.Pin;
+
+import java.util.Collections;
+import java.util.List;
+
+public class Ready extends State {
+    private static final Ready READY = new Ready();
+
+    private Ready() {}
+
+    public static Ready getInstance() {
+        return READY;
+    }
+
+    @Override
+    public State downPins(Pin downPins) {
+        if (downPins.isAllDown()) {
+            return Strike.getInstance();
+        }
+
+        return Playing.of(downPins);
+    }
+
+    @Override
+    public List<Integer> getDownPins() {
+        return Collections.EMPTY_LIST;
```

Comment:
```suggestion
        return Collections.emptyList();
```
를 사용하는게 더 좋을 것 같아요 `Collections.EMPTYLIST`는 리턴타입이 `List`라서 워닝이 뜨네요...